lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -435,6 +435,12 @@
     &:last-child {
       margin-bottom: 0;
     }
+        
+    legend {
+        // use the same styles as H3s
+        font-size: 1.2em;
+        margin-bottom: 15px;
+    }
```

Comment:
I need to find the PR that added `hd hd-n` and make it globally available. I think I scoped it at the time. Once those heading styles are available globally, we can rip this out.